lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -42,11 +44,13 @@ class TipsOrganism extends StatelessWidget {
           const SeparatorAtom(variant: SeparatorVariant.farApart),
           Center(
             child: Platform.isAndroid
-                ? ButtonAtom(
-                    variant: ButtonVariant.primary,
-                    onPressed: () => DndController.instance.toggleDnd(),
-                    text: 'dnd',
-                  )
+                ? dndEnable
+                    ? TextAtom('dnd_on')
+                    : ButtonAtom(
+                        variant: ButtonVariant.primary,
+                        onPressed: () => DndController.instance.toggleDnd(),
```

Comment:
Can be simplified to 
                        ` onPressed:DndController.instance.toggleDnd,`